src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -0,0 +1,291 @@
+package hex.genmodel.easy;
```

Comment:
Ok, so in java world the most popular way of dealing with third-party libraries is to use some dependency management tool. In this project we use Maven which is still the most popular. Long story short: it gives us a way to simply attach required libraries in a specific versions. It is also a building tool, so it takes care of attaching (if necessary) those libraries to the final artifact. No need for copying, no need for any VCS-magic :)
Classes you've copied belong to h2o-genmodel library. This library have been already attached to our project, but in a version that doesn't contain those classes. So the only thing you have to do is to update dependency version of h2o-genmodel in pom.xml. The newest version is 3.6.0.11 and it contains required classes.
After that you will be able to remove the copied code.